pkg/api/marshal.go

```diff
@@ -43,7 +43,7 @@ func marshalCommonFooter(out *errorWrapper, warnings []string) {
 }
 
 func marshalMatrixData(out *errorWrapper, data promql.Matrix) {
-	out.WriteStrings(`{"resultType":"`, parser.ValueTypeMatrix, `","result":[`)
+	out.WriteStrings(`{"resultType":"`, string(parser.ValueTypeMatrix), `","result":[`)
```

Comment:
Please don't change code independant of the PR at hand: it makes it more difficult to review PRs, and to track down where changes happen.

Reply:
This PR has two commits: 1st for changes related to just PromQL and the 2nd for changes that are dependent on the PromQL update. This line is part of the 2nd commit.